src/Raven.Client/ServerWide/Commands/TcpConnectionInfo.cs

```diff
@@ -7,15 +7,27 @@ public class TcpConnectionInfo
         public int Port;
         public string Url;
         public string Certificate;
-
+        public string[] ServerUrls;
```

Comment:
can we rename that to `Urls` the single Url property is called `Url`, do not see a good reason to not call this property `Urls`